app/views/requests/_letter_of_consent.html.erb

```diff
@@ -1,5 +1,10 @@
-<%= f.govuk_file_field :letter_of_consent, label: nil, hint: { text: simple_format(t("helpers.hint.request_form.letter_of_consent")) } %>
-<p class="govuk-body-m"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
+<span class="govuk-body">You must provide a letter of consent from the subject of this request which:</span>
+<%= govuk_list ["States that they consent to their information being released to you", "Is signed by them", "Is dated within the last 6 months"], type: :bullet %>
+<%= f.govuk_label t("helpers.label.request_form.letter_of_consent_no_template") %>
 
+<%= f.govuk_label t("request_form.letter_of_consent_title"), size: "s"  %>
+<%= f.govuk_label t("helpers.hint.request_form.letter_of_consent_file_size") %>
```

Comment:
This needs to be set as the hint and label for `letter_of_consent`, then it will automatically be shown. It shouldn't be separate HTML.